### PR TITLE
add: `steam-deb`

### PR DESCRIPTION
**Pull request by WRM-42**
- [x] Edit packagelist
 - [x] Add initial pacscript
 - [ ] ~Contact devs (via email)~
 - [x]  Add maintainer to pacscript

### CODE REVIEW

**Review by wizard-28 — CHANGES_REQUESTED** [review]
Add `breaks`

**Comment by WRM-42** [comment]
> Add `breaks`

I already added it in the past before I realized through the CI logs that the deb package would not install without Steam Launcher

File: packages/steam-deb/steam-deb.pacscript [hunk 1]
```diff
@@ -0,0 +1,8 @@
+name="steam-deb"
+gives="steam"
```
Comment (Henryws) [inline packages/steam-deb/steam-deb.pacscript:2]:
```suggestion
gives="steam"
breaks="${gives}-git ${gives}-bin ${gives}-app"
```

**Review by Henryws — CHANGES_REQUESTED** [review]

File: packages/steam-deb/steam-deb.pacscript (hunk 1 above)
Comment (WRM-42) [reply packages/steam-deb/steam-deb.pacscript:2]: I don't think Steam provides any of these

Comment (Henryws) [reply packages/steam-deb/steam-deb.pacscript:2]: If a package comes along in the future that is a binary or appimage, this is gonna be needed.

**Review by Henryws — APPROVED** [review]

File: packages/steam-deb/steam-deb.pacscript [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+version="1.0.0.74"
+pacdeps=("steam-launcher-deb")
+breaks="${gives}-git ${gives}-bin ${gives}-app"
```
Comment (wizard-28) [inline packages/steam-deb/steam-deb.pacscript:5]:
```suggestion
breaks="${gives} ${gives}-git ${gives}-bin ${gives}-app"
```

**Review by wizard-28 — CHANGES_REQUESTED** [review]

**Review by wizard-28 — APPROVED** [review]

**Pull request merged**